### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by vittoriocataffo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -243,18 +243,18 @@ $ mkdir -p autosim_nmpc
 $ git clone -b nmpc_panoc https://github.com/vittoriocataffo/A-Nonlinear-Model-Predictive-Control-Strategy-for-Autonomous-Racing-of-Scale-Vehicles.git
 $ cd PANOC_DYNAMIC_MOTOR_MODEL
 $ python3 all_wheel_drive_PANOC_DINAMYC_motor_model.py
-$ python3 PANOC_DINAMYC_motor_model.py
```
Comment (vittoriocataffo) [inline README.md:246]: "PANOC_DINAMYC_motor_model.py " is to build the NMPC with the Tatulea's parameters. I think leaving it here might be misleading

File: README.md [hunk 2]
```diff
@@ -243,18 +243,18 @@ $ mkdir -p autosim_nmpc
...
+<!--- $ python3 PANOC_DINAMYC_motor_model.py --> 
 $ cd ../OBS_PANOC_DYNAMIC_MOTOR_MODEL
 $ python3 OBS_all_wheel_drive_PANOC_DINAMYC_motor_model.py
-$ python3 OBS_PANOC_DINAMYC_motor_model.py
+<!--- $ python3 OBS_PANOC_DINAMYC_motor_model.py -->
```
Comment (vittoriocataffo) [inline README.md]: "OBS_PANOC_DINAMYC_motor_model.py" is to build the NMPC with the Tatulea's parameters. I think leaving it here might be misleading

**Comment by gsilano** [comment]
@vittoriocataffo , could you modify the README.md file following my comments? Thanks!

File: README.md [hunk 3]
```diff
@@ -243,18 +243,18 @@ $ mkdir -p autosim_nmpc
...
 
 
 The result will be a folder entitled `dynamic_my_optimizer` containing the build of the NMPC. To make everything work, it is fundamental to correct link the generated file with the scripts running of controller running along with the simulation. Therefore, att the end of the building process, you need to follow the instruction reported at terminal line. In other words,
 
-1. Edit the path provided as argument to the function `sys.path.insert` in the files entitled `OBS_all_wheel_drive_NEW_MOTOR_PANOC_my_mpc_node_DYNAMIC.py`, `NEW_MOTOR_PANOC_my_mpc_node_DYNAMIC_OBS.py`, `NEW_MOTOR_PANOC_my_mpc_node_DYNAMIC.py`and `all_wheel_drive_NEW_MOTOR_PANOC_my_mpc_node_DYNAMIC.py` contained in the `autosim_ws/src/simulator/scripts` folder. This will connect the built NMPC controller with the scripts running the simulation.
```
Comment (vittoriocataffo) [inline README.md:257]: I think that reporting at this point also the scripts to run the NMPC with the Tatulea's parameters might be misleading. I think it is better to write about them only at the end of the readme file.

**Pull request merged**